tagcompare/webdriver.py

```diff
@@ -12,42 +12,70 @@
 LOGGER = logger.Logger(name=__name__).get()
 
 
-def _read_remote_webdriver_key():
-    data = settings.DEFAULT.webdriver
-    return str.format('{}:{}', data['user'], data['key'])
-
+def setup_webdriver(capabilities=None):
+    driver = __setup_remote_webdriver(capabilities=capabilities)
+    driver.implicitly_wait(20)
+    driver.get("about:blank")
+    return driver
 
-def setup_webdriver(remote=True, capabilities=None):
-    if remote:
-        if not capabilities:
-            raise ValueError("capabilities must be defined for remote runs!")
 
-        remote_webdriver_url = "http://{}@{}".format(
-            _read_remote_webdriver_key(), settings.DEFAULT.webdriver['url'])
-        driver = webdriver.Remote(
-            command_executor=remote_webdriver_url,
-            desired_capabilities=capabilities)
-    else:
-        driver = webdriver.Firefox()
+def __setup_remote_webdriver(capabilities):
+    if not capabilities:
+        raise ValueError("capabilities must be defined for remote runs!")
+
+    # Update capabilities
+    capabilities['public'] = 'share'
+
+    user = settings.DEFAULT.webdriver['user']
+    key = settings.DEFAULT.webdriver['key']
+    remote_webdriver_url = "http://{}:{}@{}".format(
+        user, key, settings.DEFAULT.webdriver['url'])
+    driver = webdriver.Remote(
+        command_executor=remote_webdriver_url,
+        desired_capabilities=capabilities)
+    """ TODO: Remove
+    sauce = SauceClient(user, key)
+    jobs = sauce.jobs.get_jobs()
+    print jobs
+    """
 
-    driver.implicitly_wait(20)
-    driver.get("about:blank")
+    remote_url = "http://www.saucelabs.com/jobs/%s" % driver.session_id
+    LOGGER.debug("Remote webdriver URL: %s" % remote_url)
     return driver
 
 
+def check_browser_logs(driver):
+    """
+    Checks browser for errors, returns a list of errors
+    :param driver:
+    :return:
+    """
+    browserlogs = driver.get_log('browser')
+    errors = []
+    for entry in browserlogs:
+        if entry['level'] == 'SEVERE':
```

Comment:
Wonder if setting this lower to warning will provide similar value without being too verbose.